### PR TITLE
Manage use of BSS Provider by Tyr

### PR DESCRIPTION
**Pull request by pthegner**

### CODE REVIEW

**Comment by kinnou02** [comment]
You will have to modify the instance model in navitiacommon and it would be nice to had some test

File: source/tyr/tyr/resources.py [hunk 1]
```diff
@@ -366,6 +367,8 @@ def put(self, id=None, name=None):
                             location=('json', 'values'), default=instance.night_bus_filter_base_factor)
         parser.add_argument('priority', type=int, help='instance priority',
                             location=('json', 'values'), default=instance.priority)
+        parser.add_argument('bss_provider', type=bool, help='bss provider activation',
+                            location=('json', 'values'), default=True)
```
Comment (kinnou02) [inline source/tyr/tyr/resources.py]: We prefer to use the previous value as default when updating, it's allow us to do partial update

File: source/tyr/tyr/resources.py [hunk 2]
```diff
@@ -96,7 +96,8 @@ def format(self, value):
     'walking_transfer_penalty': fields.Raw,
     'night_bus_filter_max_factor': fields.Raw,
     'night_bus_filter_base_factor': fields.Raw,
-    'priority': fields.Raw
+    'priority': fields.Raw,
+    'bss_provider': fields.Boolean
```
Comment (kinnou02) [inline source/tyr/tyr/resources.py]: The same thing should be done in the status API of Jormungandr

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by buildbotctp** [comment]
Build finished.

**Comment by kinnou02** [comment]
Jenkins, test this please.

**Pull request merged**